test/plugin_functional/test_suites/doc_views/doc_views.ts

```diff
@@ -43,7 +43,7 @@ export default function ({ getService, getPageObjects }: PluginFunctionalProvide
     });
 
     it('should show custom doc views', async () => {
-      await testSubjects.click('docTableExpandToggleColumn');
+      await testSubjects.click('docTableExpandToggleColumn-0');
```

Comment:
+1, yeah looks like we added an index which imo is better for more consistent tests.

Reply:
yeah we have to. since we adopt data grid, we have `docTableExpandToggleColumn-[row index]` now.